backend/src/Request/CaptainFinancialSystem/CaptainFinancialSystemDetailRequest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace App\Request\CaptainFinancialSystem;
+
+use App\Entity\CaptainEntity;
+
+class CaptainFinancialSystemDetailRequest
+{
+    private int $id;
```

Comment:
Why is this field exist? And if it should always be int, why you don't mentioned that for the front in the OpenAPI doc?